javascript-closures/my-app/src/Timer.tsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react-hooks/exhaustive-deps -- Delete this line! */
+import { useEffect, useState } from 'react';
+
+export function Timer() {
+  let [time, setTime] = useState(0);
+  useEffect(() => {
+    const timerId = setTimeout(() => {
+      setTime(time += 1);
+      console.log(`Time is: ${time}`);
+    }, 1000);
+    return () => clearTimeout(timerId);
+  }, [setTime, time]);
```

Comment:
This one is correct, but since setTime is a useState setter, it does not have to be listed (though it can be).